docs/Tutorial.md

```diff
@@ -136,22 +136,40 @@ I think it's still free for students with a school email.
 
 ## ZModeler3
 
-I don't want to talk about it for very long.
+Zmod is vital piece of software for creating any type of vehicles within GTA. The main things you need to know about zmod are:
```

Comment:
> The main things you need to know about zmod are:

Let's make this "Some helpful tips for zmod include:" as the content below this is more tips than an overview of the software.